src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,53 @@
+.watch {
+  height: 80vw;
+  max-height: 80vh;
+  width: 80vw;
+  max-width: 80vh;
+  border: 5px dashed #000;
+  border-radius: 50%;
+  position: relative;
+
+  &__hand {
+
+    &--minute {
+      @extend %absolute-centering;
+      animation: minute 5s linear infinite;
+    }
+
+    &--hour {
+      @extend %absolute-centering;
+      animation: hour 60s linear infinite;
+    }
+
+  }
+
+  &__center {
+    height: 6vw;
+    max-height: 6vh;
+    width: 6vw;
+    max-width: 6vh;
+    @extend %absolute-centering;
+    background-color: yellowgreen;
+    border-radius: 50%;
+    z-index: 1;
+  }
+
+}
+
+@keyframes hour {
+  from {
+    transform: translate(-50%,-50%) rotate(0deg);
+  }
+  to {
+    transform: translate(-50%,-50%) rotate(360deg);
+  }
+}
+
+@keyframes minute {
+  from {
+    transform: translate(-50%,-50%) rotate(0deg);
+  }
+  to {
+    transform: translate(-50%,-50%) rotate(360deg);
+  }
+}
```

Comment:
this animations is completely the same as `hour `, so try not repeat yourself and use one animation for both hands